docker/Dockerfile.omp_offload

```diff
@@ -0,0 +1,110 @@
+##############################################################
+# This Dockerfile contains the additional NVIDIA compilers, 
+# libraries, and plugins to enable OpenMP offloading and NVIDIA GPU 
+# acceleration of Devito codes.
+#
+# TOFIX: Uncomment line "self.cflags += ['-Xopenmp-target', '-march=sm_37']" in devito/arch/compiler.py and change sm_37 for sm_XX where XX is the compute capability of your GPU
+# BUILD: 
+#   sudo docker build -t devito/ompoffload:1.0 -f docker/Dockerfile.omp_offload .
+# 
+# RUN
+#   sudo docker run --gpus all --rm -it devito/ompoffload:1.0
+# python examples/seismic/acoustic/acoustic_example.py
+
+FROM nvidia/cuda:10.1-devel-ubuntu18.04
+
+ENV DEBIAN_FRONTEND noninteractive
+CMD ["/bin/bash"]
+
+## INSTALL PREREQUISITES
+
+RUN apt-get update
+RUN apt-get install -y --no-install-recommends apt-utils
+RUN apt-get install -y --no-install-recommends build-essential
+RUN apt-get install -y --no-install-recommends libelf-dev
+RUN apt-get install -y --no-install-recommends libffi-dev
+RUN apt-get install -y --no-install-recommends pkg-config
+RUN apt-get install -y --no-install-recommends libnuma-dev
+RUN apt-get install -y --no-install-recommends gcc-multilib
+RUN apt-get install -y --no-install-recommends g++-multilib
+RUN apt-get install -y --no-install-recommends wget
+# latest cmake
```

Comment:
IS this really necessary? What's wrong with the default cmake3.

Reply:
Also it's usually safer to just install from source that use these unregistered repository.